tests/tabnote_tests.ts

```diff
@@ -4,6 +4,7 @@
 // TabNote Tests
 
 import { Flow } from 'flow';
+// import { Font, FontWeight } from 'font'; // RONYEH
```

Comment:
You could leave this file unmodified, `Font.SANS_SERIF`is already merged producing no visual diff.